.coveragerc

```diff
@@ -4,7 +4,10 @@ omit =
 
 [report]
 omit =
+  cnxdb/contrib/pytest.py
+  cnxdb/contrib/testing.py
```

Comment:
Hmm, that page doesn't give much reasoning, but this: https://jacobian.org/writing/what-is-django-contrib/ does. I like the "Good contrib packages tackle issues that that are not trivial, are bikeshed prone, and are difficult to get right for the common case." And test framework for DB falls into that box nicely. I bet that contrib started as "stuff people dumped on us that we don't want in core" though -  the name isn't very semantically useful in and of itself. But we can go with this.